src/mate/academy/hashMap/Main.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.hashMap;
+
+public class Main {
+    public static void main(String[] args) {
+        ReferenceMyHashMap<String, String> myHashMap = new ReferenceMyHashMap<>();
+        myHashMap.put("1", "2");
```

Comment:
Verify case when add 2 entries with same key

Reply:
@Siri-O I checked this, it overwrites the value